snippets/csharp/VS_Snippets_CFX/c_customauthpol/cs/c_customauthpol.cs

```diff
@@ -67,10 +67,12 @@ public bool Evaluate(EvaluationContext evaluationContext, ref object state)
 		state = customstate;
 	  }
 	  else
-		customstate = (CustomAuthState)state;
+            {
+                customstate = (CustomAuthState)state;
+            }
 
```

Comment:
Indentation needs to be fixed.

Reply:
To keep this PR simple, I'm not making extra changes.